src/org/opendatakit/briefcase/export/FormDefinition.java

```diff
@@ -144,7 +150,25 @@ public static FormDefinition from(BriefcaseFormDefinition briefcaseFormDefinitio
   }
 
   private static Map<String, QuestionDef> getFormControls(FormDef formDef) {
-    formDef.getEvaluationContext().addFunctionHandler(DUMMY_PULLDATA_HANDLER);
+    Pattern p = Pattern.compile("^.+cannot handle function '(.+?)'.*$");
+    List<IFunctionHandler> customFunctionHandlers = getAllDescendants(formDef.getMainInstance().getRoot())
+        .stream()
+        .flatMap(e -> {
+          try {
+            formDef.triggerTriggerables(e.getRef(), true);
+            return Stream.empty();
+          } catch (Throwable t) {
+            return Stream.of(t.getMessage().split("\n"));
```

Comment:
JavaRosa will create a message with two lines. Example: 
```
Exception in thread "Thread-4" java.lang.RuntimeException: Error evaluating field 'firstname': The problem was located in calculate expression for /FSQ/firstname
XPath evaluation: cannot handle function 'trim'
```

We could go for the function name directly to the second line, but then we need to account for other unexpected exceptions that could have less than two lines, etc.

This implementation is more abstract with the tradeoff of having to process more data, which feels fine.